presto-main/src/main/java/com/facebook/presto/sql/planner/ExpressionExtractor.java

```diff
@@ -63,6 +66,14 @@ protected Void visitPlan(PlanNode node, ImmutableList.Builder<Expression> contex
         }
 
         @Override
+        public Void visitAggregation(AggregationNode node, ImmutableList.Builder<Expression> context)
```

Comment:
@martint What do you think about this proposal?

Regardless whether we want to refactor the code, this commit is ok to merge because refactor commits should not be mixed with other stuff.